python/pyspark/ml/tests.py

```diff
@@ -1075,7 +1076,8 @@ def test_linear_regression_summary(self):
         pValues = s.pValues
         self.assertTrue(isinstance(pValues, list) and isinstance(pValues[0], float))
         # test evaluation (with training dataset) produces a summary with same values
-        # one check is enough to verify a summary is returned, Scala version runs full test
+        # one check is enough to verify a summary is returned
+        # The child class LinearRegressionTrainingSummary runs full test
```

Comment:
@MLnick This is because the type of ```LinearRegressionModel.summary``` is ```LinearRegressionTrainingSummary```, but the return type of ```LinearRegressionModel.evalute()``` is ```LinearRegressionSummary```. Theoretically we should test both, but we can simplify the test for only check one function since we have checked all functions in the child class. Thanks for @mpjlu to update this comments.